<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zJointLabelling</class>
 <widget class="QDialog" name="zJointLabelling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Joint Labelling</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
   <item row="0" column="0">
    <widget class="QWidget" name="mainBox" native="true">
     <layout class="QVBoxLayout" name="mainLayout" stretch="0,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="MainButtonLayouts">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="Layout1">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="Layout1Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Center</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Root">
            <property name="text">
             <string>Root</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Spine">
            <property name="text">
             <string>Spine</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Neck">
            <property name="text">
             <string>Neck</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Head">
            <property name="text">
             <string>Head</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="NoLabel">
            <property name="text">
             <string>Remove label</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Layout2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="Layout2Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Arm</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Collar">
            <property name="text">
             <string>Collar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Shoulder">
            <property name="text">
             <string>Shoulder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Elbow">
            <property name="text">
             <string>Elbow</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Hand">
            <property name="text">
             <string>Hand</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Finger">
            <property name="text">
             <string>Finger</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Layout3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="Layout3Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Leg</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Hip">
            <property name="text">
             <string>Hip</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Knee">
            <property name="text">
             <string>Knee</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Foot">
            <property name="text">
             <string>Foot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Toe">
            <property name="text">
             <string>Toe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="PropLabel">
            <property name="text">
             <string>Props</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="PropLayout">
            <item>
             <widget class="QToolButton" name="PropA">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="PropB">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="PropC">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Layout4">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="Layout4Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Fingers</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Thumb">
            <property name="text">
             <string>Thumb</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Index">
            <property name="text">
             <string>Index</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Middle">
            <property name="text">
             <string>Middle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Ring">
            <property name="text">
             <string>Ring</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Pinky">
            <property name="text">
             <string>Pinky</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ExtraFinger">
            <property name="text">
             <string>Extra</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Layout5">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="Layout5Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Toes</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BigToe">
            <property name="text">
             <string>Big toe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="IndexToe">
            <property name="text">
             <string>Index toe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MiddleToe">
            <property name="text">
             <string>Middle toe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RingToe">
            <property name="text">
             <string>Ring toe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PinkyToe">
            <property name="text">
             <string>Pinky toe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ExtraToe">
            <property name="text">
             <string>Extra</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="OtherLabel">
        <property name="text">
         <string>Enter a joint label, or &quot;label#&quot; to label incrementally</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="otherLayout">
        <item>
         <widget class="QLineEdit" name="otherTextEntry"/>
        </item>
        <item>
         <widget class="QToolButton" name="otherTextSet">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="SideLayout">
        <item>
         <widget class="QLabel" name="sideLabel">
          <property name="text">
           <string>Side:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="guessSide">
          <property name="text">
           <string>Guess</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="leftSide">
          <property name="text">
           <string>Left side</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="centerSide">
          <property name="text">
           <string>Center</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rightSide">
          <property name="text">
           <string>Right side</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="noSide">
          <property name="text">
           <string>No side</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qt_resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
